glitch/analysis/rules.py

```diff
@@ -15,7 +15,10 @@ class Error():
             'sec_hard_user': "Hard-coded user - Developers should not reveal sensitive information in the source code. (CWE-798)",
             'sec_invalid_bind': "Invalid IP address binding - Binding to the address 0.0.0.0 allows connections from every possible network which might be a security issues. (CWE-284)",
             'sec_no_int_check': "No integrity check - The content of files downloaded from the internet should be checked. (CWE-353)",
-            'sec_no_default_switch': "Missing default case statement - Not handling every possible input combination might allow an attacker to trigger an error for an unhandled value. (CWE-478)"
+            'sec_no_default_switch': "Missing default case statement - Not handling every possible input combination might allow an attacker to trigger an error for an unhandled value. (CWE-478)",
+            'sec_non_official_image': "Use of non-official Docker image - Use of non-official images should be avoided or taken into careful consideration.",
```

Comment:
No

Reply:
https://cwe.mitre.org/data/definitions/829.html

What about this one?

Reply:
Seems like it is a good CWE for the smell, since this smell was taken from a study which attributed a CWE to other and not to this I will have to investigate more about it